Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs

```diff
@@ -3723,7 +3724,15 @@ private void SwitchFrame(ElementInfo EI)
                 string[] iframesPathes = spliter.Split(EI.Path);
                 foreach (string iframePath in iframesPathes)
                 {
-                    Driver.SwitchTo().Frame(Driver.FindElement(By.XPath(iframePath)));
+                    try
+                    {
+                        Driver.SwitchTo().Frame(Driver.FindElement(By.XPath(iframePath)));
+                    }
+                    catch (NoSuchElementException)
+                    {
+                        EI.Locators.ForEach((locator) => locator.LocateStatus = ElementLocator.eLocateStatus.Failed);
```

Comment:
Mark only specific locator as failed which was tested instead of marking all locators as failed